### ISSUE TITLE
Add a --install-export-templates option to the godot command line

### ISSUE THREAD

**Issue by touilleMan** [godotengine/godot-proposals#1277]
**Describe the project you are working on:**

[Godot-Python](https://github.com/touilleMan/godot-python)

**Describe the problem or limitation you are having in your project:**

I'm working on the export plugin for Godot-Python project, this is a non-trivial part of the project that need to have unit tests.

However the export system requires the export templates to be installed, this is something that is done through the GUI of the Godot editor, so it cannot be easily automated which is especially important when running the tests on a CI environement.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**

There should be an option on the Godot command line to ensure the export templates for the currently running version of Godot are installed and if it's not the case download&install them automatically.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**

```shell
$ godot --install-export-templates
$ godot --path my_project --export windows-preset export_output
```

**Comment by Calinou** [comment]
I remember talking about this very CLI option in the main Godot repository last year :slightly_smiling_face: (I can't find where I mentioned it though).

**Comment by jonbonazza** [comment]
Ive made these available:
https://hub.docker.com/repository/docker/summerplay/godot

You may be able to use the headless version (which comes with templates)

Right now the Dockerfiles that generate these container images just download the binaries from tx family servers, but i plan to make them do actual builds in the future when i have time

(Dockerfiles here for reference: https://github.com/summerplaygames/godot-docker

**Comment by umarcor** [comment]
@jonbonazza, @touilleMan, you might be interested in #1411.

**Comment by Calinou** [comment]
WIP branch: https://github.com/Calinou/godot/tree/add-install-export-templates-cli-argument

Having https://github.com/godotengine/godot/pull/35568 merged first would make this easier.

**Comment by AdrKacz** [comment]
@touilleMan do you have any updates regarding your issue? I don't see `--install-export-templates` part of the `--help` of Godot 4, but it seems it can executes.

**Comment by Calinou** [comment]
> I don't see --install-export-templates part of the --help of Godot 4, but it seems it can executes.

Godot doesn't warn when using unknown CLI arguments, as the current project may be parsing them. There is no such CLI argument currently implemented.

**Comment by AThousandShips** [comment]
They are not, this has not been implemented, see the command line arguments with `--help`

**Comment by HarperCT** [comment]
++ to any godot devs watching. Would be great to avoid having to do some hacky workarounds in our CI pipelines :)

**Comment by felipemullen** [comment]
Goodness, issue has been open since 2020. 
Any time I expect some modern solution to a problem here in Godot land, I find some nearly decade old issue.

I really Captain Juan would open up the platform to community governing. The pace is so slow when we have to rely on a handful of gents to manually review each PR.

Sigh. Off to some hacky CI workaround.

**Comment by Calinou** [comment]
@felipemullen Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.

**Comment by felipemullen** [comment]
@Calinou fair enough, my intent is not to disrupt. But at some point if you can find a way to relay the message to someone who listens:
There are probably thousands of developers out there who are willing to contribute to Godot but get discouraged by the number of dead PRs in the main repo. I am one of those developers.
